networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py

```diff
@@ -635,13 +635,27 @@ def _setup_default_infrastructure_rules(self):
             path = API.POLICY.format(policy["id"])
             res = self.client.get(path=path)
             if res.ok:
-                continue
+                if self._check_for_infrastructure_changes():
```

Comment:
What is this method? I haven't found it in the source. Do you want to call `_check_infrastructure_rules_for_updates()` here?